CRM/Event/Tokens.php

```diff
@@ -60,22 +60,86 @@ protected function getBespokeTokens(): array {
         'data_type' => 'String',
         'audience' => 'user',
       ],
-      'contact_email' => [
+      'loc_block_id.email_id.email' => [
         'title' => ts('Event Contact Email'),
-        'name' => 'contact_email',
+        'name' => 'loc_block_id.email_id.email',
         'type' => 'calculated',
         'options' => NULL,
         'data_type' => 'String',
         'audience' => 'user',
       ],
-      'contact_phone' => [
+      'loc_block_id.email_2_id.email' => [
+        'title' => ts('Event Contact Email 2'),
+        'name' => 'loc_block_id.email_2_id.email',
+        'type' => 'calculated',
+        'options' => NULL,
+        'data_type' => 'String',
+        'audience' => 'sysadmin',
```

Comment:
is there a good reason why email 1 is user and email 2 is sysadmin?

Reply:
well it's just that people get upset if we put too many tokens in the drop down cos it gets cluttered / easy to pick the wrong one